New version: DynamicOED v0.1.1

JuliaRegistrator opened the pull request:
- Registering package: DynamicOED
- Repository: https://github.com/mathopt/DynamicOED.jl
- Created by: @AlCap23
- Version: v0.1.1
- Commit: 8b7c3931026b693f446f55b17179ae4792b2105f
- Reviewed by: @AlCap23
- Reference: https://github.com/mathopt/DynamicOED.jl/issues/12#issuecomment-1999542623
- Description: Optimal experimental design of ODE and DAE systems in julia
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598d717c96cc9d7a4e18ab9d97a0801189b3bec6d65772fcec3860b75e84233bade90c3ae9514b511aef0a6ae7bc00da422981a5a3a3ee387cb083a22e4ee946bee8d076e1bce9bc6dd282b59c5854fea4cab1374f939d3b9aca654f762a58adaeefe640aac1bd5268e5bb4de6713e5135187153d6de6c44334b9e17dfdcc218a5b -->

AlCap23 commented:
[noblock] 

This is basically a fix before working on the ModelingToolkit 9 release. I would appreciate a merge just as a fix before breaking happens. Thanks!

JuliaRegistrator committed:
New version: DynamicOED v0.1.1

UUID: bec7fea8-67ad-492e-b01d-b4dfa914b0da
Repo: https://github.com/mathopt/DynamicOED.jl.git
Tree: b116ec01e375534467a17d5e61becd9e413ebf96

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
